docs/getting-started.md

```diff
@@ -178,6 +178,120 @@ $ curl -i localhost:8080/httpbin/get
 }
 ```
 
+### Kubernetes Service as an Upstream Target
+
+Tyk Operator allows accessing your Kubernetes service as an upstream proxy target.
+Thus, you can set the `proxy.target_url` as a Kubernetes Service following [DNS for Services and Pods guideline](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/), 
+so that the requests will be proxied to your service.
+
+> In general, Kubernetes Services have `<service-name>.<namespace-name>.svc.cluster.local` DNS entry once they are created. 
+For example, if you have a service called `httpbin` in `default` namespace, you can contact `httpbin` service with 
+`httpbin.default.svc` DNS record in the cluster, instead of IP addresses. 
+Please visit the official [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) for more details.
+
+Suppose we want to create a Deployment of popular [`httpbin` service](https://hub.docker.com/r/kennethreitz/httpbin/) 
```

Comment:
I think correct is "the popular" and without syntax highlighting on the "service" word